specification/components/schemas/path/BookingId.yaml

```diff
@@ -0,0 +1,9 @@
+name: id
+in: path
+description: |
+    The ID of the booking to be supplied. It must be valid and relate to an existing Booking Id otherwise a HTTP 404 response will be returned.
```

Comment:
We should use the API terminology where ever possible here I think, so we should use "Appointment" as opposed to "Booking".